cbdr/cortex
# COR-772: Update Cortex ES to 5.x

## Purpose:
See #416

This sets the stage and makes the necessary changes to update Cortex's elastic search from 2.4.1 to 5.4.0. Changes in this PR include:

* Bump Cortex Plugin to new version for Beta Cortex ES compatibility
* Update `publish_filter` ES query to work with 5.4.0 syntax
* Remove pesky deprecation warnings

> Before we start building our wonderful mappings, queries, filters and sorts for Beta Cortex, we need to upgrade ElasticSearch from 2.x to 5.x, so that we don't have to reinvent these queries in 6 months when we're forced to upgrade for some feature we need.

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-734

## Steps to Take On Prod
* Go through our API Endpoints and all pages in Legacy
* Confirm that the page isn't failing
* Confirm that the data matches expectation

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * Prepares ES to update from 2.4.1 to 5.4.0
  * Bumps `cortex-plugins-core` to 0.12.4

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-6.development.c66.me

## How to Verify These Changes
* Specific pages to visit
  * All API endpoints
  * Literally every page in Legacy

* Steps to take
  * Go through our API Endpoints
  * Confirm that the page isn't failing
  * Confirm that the data matches expectation

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
